lib/vagrant-libvirt/action/create_domain.rb

```diff
@@ -167,6 +167,19 @@ def call(env)
 
             disk[:absolute_path] = storage_prefix + disk[:path]
 
+            if not disk[:pool].nil?
+              disk_pool_name = disk[:pool]
```

Comment:
This is a local variable while below you use an instance variable, might be best to use local variable here only for the disk_pool_name. Mixing may result in some weird behaviour on subsequent loops:

the following prints what is expected
```
ruby -e 'def test
  value = "a different value"
  @value = "some value"
  puts @value
end

test()'
some value
```

This does not though and it could occur for a second or third disk where the pool isn't set.
```
ruby -e 'def test
  @value = "a different value"
  value = "some value"
  puts @value
end

test()'
a different value
```